apps/balancer-tools/src/app/poolsimulator/(components)/AnalysisPoolParamsForm.tsx

```diff
@@ -0,0 +1,78 @@
+"use client";
+
+import { useEffect } from "react";
+import { useForm } from "react-hook-form";
+
+import { Select, SelectItem } from "#/components/Select";
+import { Form, FormLabel } from "#/components/ui/form";
+import { usePoolSimulator } from "#/contexts/PoolSimulatorContext";
+
+export default function AnalysisPoolParamsForm() {
+  const {
+    initialData,
+    customData,
+    setAnalysisTokenByIndex,
+    analysisToken,
+    setCurrentTabTokenByIndex,
+  } = usePoolSimulator();
+
+  const form = useForm({
+    mode: "onChange",
+  });
+
+  const {
+    setValue,
+    getValues,
+    formState: { errors },
+  } = form;
+
+  const onSubmit = () => {
+    if (Object.keys(errors).length) return;
+  };
```

Comment:
👀 there's something missing here, no?

Reply:
not really... I just meant to add the sidebar with the token selector and used what we already had on StableSwap. But for this use case, we don't even need a form rn 😆 will remove it